website_legal_page/controllers/main.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2020 Alexandre Díaz - Tecnativa
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import http
+from odoo.http import request
+
+
+class WebsiteLegalPage(http.Controller):
+    @http.route(["/shop/terms"], type="http", auth="public", website=True)
```

Comment:
In 15.0 this is /terms not /shop/terms
![image](https://user-images.githubusercontent.com/10028499/157883619-8ee11f75-0df0-47c2-b4e1-0e06d41ac034.png)

Reply:
This seems to be pending.